libsigopt/views/rest/spe_next_points.py

```diff
@@ -205,7 +207,7 @@ def draw_samples(
     max_value = sigopt_parzen_estimator.evaluate_expected_improvement(numpy.atleast_2d(max_location))[2][0]
 
     uniform_domain = deepcopy(domain.one_hot_domain)
-    uniform_domain.set_quasi_random_sampler_opts(sampler="uniform")
+    uniform_domain.set_quasi_random_sampler_opts({"sampler": "uniform"})
```

Comment:
`SamplerOpts(sampler="uniform")` ?

Reply:
Effectively the same thing, mypy is smart enough to check the type for the input dictionary. But I can explicitly make it a SamplerOpt too